packages/OreIdLoginButton/package.json

```diff
@@ -26,7 +27,7 @@
     "react-dom": "^16.8.0 || ^17.0.0|| ^18.0.0"
   },
   "devDependencies": {
-    "rollup-plugin-terser": "^7.0.2",
+    "@aikon/aikon-configs": "^1.0.5",
```

Comment:
This is a public repo. aikon-configs is private and part of our monorepo so I don't think we should add this dependency. This is only here for the prettier config right? We should just copy it. It's really small and doesn't change.

Reply:
Aikon-configs should be public. It has only lint and prettier rules...
There is no confidential information of any kind, just things to format the code.

Reply:
At least the (published) package should be public

Reply:
Ok, I didn't realize it was published as public.

Reply:
I don't know if it's public. I said it should (and there's no point in not be)

Reply:
I checked, it is published publicly